### PR TITLE
wait for a moment before start to clean up yurt-e2e-test namespace

### LINKED ISSUE #1054: autonomy test failed for deleting namespace
#### Which jobs are failing:

When cleaning the test environment after all tests pass, e2e tests sometimes fail for deleting test namespace. The error log is like this:
```
[AfterSuite] [FAILED] [0.185 seconds]
[AfterSuite] 
/home/runner/work/openyurt/openyurt/test/e2e/autonomy/autonomy_test.go:251

  Begin Captured GinkgoWriter Output >>
    STEP: delete namespace:yurt-e2e-test 11/09/22 06:59:13.458
  << End Captured GinkgoWriter Output

  fail to delete created namespaces:yurt-e2e-test
  Unexpected error:
      <*url.Error | 0xc000550c90>: {
          Op: "Delete",
          URL: "https://127.0.0.1:39939/api/v1/namespaces/yurt-e2e-test",
          Err: <*net.OpError | 0xc000386820>{
              Op: "read",
              Net: "tcp",
              Source: <*net.TCPAddr | 0xc000550ab0>{IP: [127, 0, 0, 1], Port: 48720, Zone: ""},
              Addr: <*net.TCPAddr | 0xc000550ae0>{IP: [127, 0, 0, 1], Port: 39939, Zone: ""},
              Err: <*os.SyscallError | 0xc000558700>{
                  Syscall: "read",
                  Err: <syscall.Errno>0x68,
              },
          },
      }
      Delete "https://127.0.0.1:39939/api/v1/namespaces/yurt-e2e-test": read tcp 127.0.0.1:48720->127.0.0.1:39939: read: connection reset by peer
  occurred
  In [AfterSuite] at: /home/runner/work/openyurt/openyurt/test/e2e/common/ns/ns.go:37
```

It seems that the connection was closed by APIServer. I have not figured out the reason why it is reset. 

Sometimes it can pass: 
https://github.com/openyurtio/openyurt/actions/runs/3425939958/jobs/5707228507

,and sometimes cannot:
https://github.com/openyurtio/openyurt/actions/runs/3425905272/jobs/5707158117

#### labels

/kind failing-test

### PR DESCRIPTION
**Pull request by YTGhost**
Signed-off-by: HIHIA <283304489@qq.com>

#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:
After https://github.com/openyurtio/openyurt/pull/1060, 10s doesn't seem to be enough.

#### Which issue(s) this PR fixes:

Fixes #1054 

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: test/e2e/autonomy/autonomy_test.go [hunk 1]
```diff
@@ -258,7 +258,7 @@ var _ = ginkgo.AfterSuite(func() {
 	ginkgo.By("delete namespace:" + YurtE2ENamespaceName)
 	gomega.Eventually(func() error {
 		return ns.DeleteNameSpace(c, YurtE2ENamespaceName)
-	}).WithTimeout(10 * time.Second).WithPolling(1 * time.Second).Should(gomega.Succeed())
+	}).WithTimeout(20 * time.Second).WithPolling(1 * time.Second).Should(gomega.Succeed())
```
Comment (Congrool) [inline test/e2e/autonomy/autonomy_test.go:261]: It seems that ns.DeleteNameSpace already has its own timeout which is 5min. Timeout we set here may have conflict with the original timeout.

Could we split the timeout as `delete ns timeout` and `reconnect timeout`, and only set `reconnect timeout` here?

I think we can use something like `clientset.Discovery().ServerVersion()` to detect if APIServer has reconnected.

**Comment by YTGhost** [comment]
@Congrool PTAL

**Comment by Congrool** [comment]
/lgtm
Thanks

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**